widget/textarea.go

```diff
@@ -590,56 +590,27 @@ func (l *TextArea) initWidget() {
 }
 
 func (l *TextArea) PrependText(value string) {
-	l.text.Label = value + l.text.Label
-
-	if l.showHorizontalSlider {
-		if l.horizontalScrollMode == ScrollBeginning {
-			l.hSlider.Current = 0
-		} else if l.horizontalScrollMode == ScrollEnd {
-			l.hSlider.Current = l.hSlider.Max
-		}
-	}
-	if l.showVerticalSlider {
-		if l.verticalScrollMode == ScrollBeginning {
-			l.vSlider.Current = 0
-		} else if l.verticalScrollMode == ScrollEnd {
-			l.vSlider.Current = l.vSlider.Max
-		}
-	}
+	l.init.Do()
+	l.SetText(value + l.text.Label)
```

Comment:
Alternatively I can extract a `setText` method and call it from the 3 exported variants.